src/createUrl.js

```diff
@@ -17,7 +17,12 @@
  * @return {string} - created URL
  */
 function createUrl(template, params) {
-  // write code here
+  const url = template.replace(/{(\w+)}/g,
+    function(object, unit) {
+      return params[unit];
+    });
```

Comment:
```suggestion
  const url = template.replace(/{(\w+)}/g, function(object, unit) {
    return params[unit];
  });
```
https://learn.javascript.ru/arrow-functions-basics